docs/bookstore/page-0/page-0-intro.md

```diff
@@ -0,0 +1,163 @@
+
+7. **Connect with External Services/API**: Learn how to connect your application with external services and APIs using Knative Eventing and Apache Camel-K integration.
+
+## **How to properly learn?**
+
+### Preview the Final Bookstore: 1-Minute Demo Video
+
+To help you visualize what you'll be creating, we've prepared a brief demo video. This two-minute preview showcases the final bookstore application, providing a clearer understanding of what you can expect to build.
+
+![Demo Video](images/7.png)
+
+[EditedV2 - Knative Bookstore Demo.mp4](https://drive.google.com/open?id=1nJGQOM9X1Wb5eKjdq-Iv-cQK97LYTscD&usp=drive_copy), Music used under [license](https://docs.google.com/document/d/125Wok7L5lzf1XnsM5HIM8SuZ1ty2NCbddKrEtM_Xebs/edit#heading=h.79x5qqa9u7gd)
+
+<iframe src="https://drive.google.com/open?id=1nJGQOM9X1Wb5eKjdq-Iv-cQK97LYTscD&usp=drive_copy" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+
+### Getting Started with the Setup Script
+
+![Setup Script](images/8.png)
+
+We recommend beginning by using our setup script to launch the bookstore application. This allows you to see what the final product will look like and gives you a chance to explore its features firsthand. Familiarizing yourself with the application upfront will make the learning process smoother and more intuitive. It is located at `solution/setup.sh`.
+
+### Step-by-Step Guidance for Beginners
+
+![Beginner Guide](images/9.png)
+
+This tutorial is meticulously structured to be beginner-friendly, featuring detailed, step-by-step instructions for building the sample app. Simply follow the sequence we've laid out: each section builds upon the last, guiding you through the construction of the application. Should you encounter any hurdles, the Knative community is a fantastic resource for support. Don't hesitate to ask questions and seek advice there.
+
+### Accelerated Learning Path for Advanced Users
+
+![Advanced Users Guide](images/10.png)
+
+If you find the tutorial too basic, or if it covers familiar territory, feel free to adjust your learning approach. Each section of the tutorial is accompanied by concise graphics that summarize key concepts. Advanced learners can choose to focus on these graphics to grasp the essentials faster, streamlining the learning experience without sacrificing depth or understanding.
+
+## **Help**
+
+![Help Image](images/11.png)
+
+Join the supportive Knative community via the Cloud Native Computing Foundation (CNCF) Slack, particularly the #knative channel. Before posting your questions, please search to see if they've already been answered. Your feedback on this tutorial is invaluable, so don't hesitate to reach out with suggestions or questions.
+
+## **Next Step**
+
+![Next Step Image](images/12.png)
+
+Clone the repo first
+```sh
+git clone git@github.com:knative/docs.git
```

Comment:
Currently receiving 2 complaints about this: ssh pull doesn't work for them.

Reply:
Maybe add a little warning letting them know they need to configure git for ssh first? Or a link to https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors ?